modin/pandas/test/test_groupby.py

```diff
@@ -1367,16 +1365,14 @@ def test_agg_exceptions(operation):
     "kwargs",
     [
         {
-            "Max": ("cnt", np.max),
-            "Sum": ("cnt", np.sum),
-            "Num": ("c", pd.Series.nunique),
```

Comment:
Why was this removed?

Reply:
test frame doesn't have any column named `"cnt"`, so previously this test was useless. It was just testing that modin and pandas will raise a `KeyError`, now this test works properly.

I think that I made these changes when I was debugging the new behavior of `groupby`, so it seems this is not correlated with updating the pandas version. Should we fix it in a different PR, or it's acceptable to keep it here?

Reply:
We may want to revert and discuss in a different PR. It is useful to check for the `KeyError`.

Reply:
What I mean is that we may want both, with the `KeyError` and with these changes. A new issue/PR is probably best for a discussion like this.

Reply:
I've reverted changes at `test_groupby` and also disabled this test. It revealed, that because of another bug, the dictionary for aggregate is empty in this test, which behavior was changed in 1.2.0, it's unclear was it intended or not, so I've asked a [question](https://github.com/pandas-dev/pandas/issues/39609) at pandas repo.

I've also created an issue about our groupby tests to add reverted changes later (#2698)